playground/fibonacci_using_for.js

```diff
@@ -0,0 +1,26 @@
+function fibo (idx) {
+    let sumNum = 0;
+    
+        if (idx == 1 || idx == 2) {
+            return 1
+        } else if( idx >= 3) {
+            let curNum=1 ;
+            let bfNum=1 ;
+            for ( let i=1; i<=idx-2; i++){
```

Comment:
🔥 사실 이 코멘트 말고는 다 린트(lint)용 코멘트

이런 리뷰가 진짜 중요한 리뷰죠.

```suggestion
            for ( let i=3; i<=idx; i++){
```

이렇게 해주시면, 실제 피보나치 idx 와 동일한 값으로 돌아갈 수 있을 것 같습니다. 이해 되시죠? 

또한 린트적으로도 연산자 양 옆으로는 보통 스페이스를 삽입합니다. ex) 1+3 (X), 1 + 3 (O)
단 단항연산자(++, --, ! 등)은 변수와 붙여서 사용하는데 잘 쓰셨습니다. 

```suggestion
            for (let i = 1; i <= idx-2; i++) {
```

Reply:
린트가 뭔가요?